CONTRIBUTING.rst

```diff
@@ -320,69 +322,92 @@ The build status for pyiron and all sub packages are given below
     :target: https://ci.appveyor.com/project/pyiron-runner/pyiron/branch/master
     :alt: Build status
 
-pyiron releases
+Pyiron release distribution
 ---------------
 
 .. image:: https://anaconda.org/conda-forge/pyiron/badges/downloads.svg
     :target: https://anaconda.org/conda-forge/pyiron/
     :alt: Downloads
 
-For the pyiron release management we use git tags::
+Pyiron is released through `conda-forge`_ and  `pip`_. 
+Both packages are created automatically and maintained with every new release of pyiron_base. In order to use these distributions simply use the following command for conda::
 
-   https://git-scm.com/book/en/v2/Git-Basics-Tagging
+   conda install -c conda-forge pyiron_base
 
-The tag format consists of a tag_prefix (<package name>-) and the release version, for example::
+In order to use the pip distribution use::
 
-   pyiron-0.2.0
+   pip install pyiron-base
 
-For the automated versioning we use::
+Just like each other commit to the master branch the tagged releases are pushed to pypi.org (https://pypi.org/project/pyiron-base/#history) and anaconda.org (https://anaconda.org/pyiron/pyiron_base)::
 
-   https://github.com/warner/python-versioneer/
+The major difference for pypi (pip) is that tagged releases are the default for pip while installing prerelease versions using pip requires the `--pre` flag::
 
-So the configuration of the release is included in setup.cfg::
+   pip install --pre pyiron
 
-   https://github.com/pyiron/pyiron_base/blob/master/setup.cfg
+Those pre-release versions are named `<version_number>.post0.dev<release number>` ::
 
-As the pyiron packages are pure python packages – we use only the Linux Python 3.7 job to build the packages, as defined in the .travis.yml file::
+   0.2.0.post0.dev1
 
-   https://github.com/pyiron/pyiron_base/blob/master/.travis.yml
+For anaconda the prereleases are pushed to the pyiron channel and can be installed using:
+conda install -c pyiron pyiron
```

Comment:
This is actually not the case. Pre-releases are not available via conda. I will have a full review later, but so far everything else looks good! Thanks!

Reply:
Ok, then I will just delete this line